.circleci/config.yml

```diff
@@ -188,19 +188,11 @@ jobs:
       - run:
           name: Run migrations ci
           command: yarn run db:migrate:ci
-      - run:
-          name: Install Headless Chrome dependencies
-          command: |
-            sudo apt-get install -yq \
-            gconf-service libasound2 libatk1.0-0 libatk-bridge2.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
-            libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \
-            libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 \
-            libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates \
-            fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
       - run:
           name: Run cucumber
           command: |
             mkdir reports
+            sleep 5
```

Comment:
Now that the headless chrome dependencies are no longer installed, we need to add a little bit of a delay.

Reply:
Why do we need the sleep here? Is it to wait for the server to spin up?

Reply:
Right. Without the delay we are getting errors since the front end server hasn't completed the startup. Before, we didn't notice it since the install of dependencies gave the server enough time.

Reply:
Oh I missed the `background: true` in the `Start Server` step! Are we open the same issue running the migrations before `sequelize` has been installed?

Reply:
We haven't had problems with migrations in the pipeline when testing the backend without starting the servers for cucumber testing. Perhaps we don't need `yarn deps`. Removing.

Reply:
I guess if we run into that issue we can move `yarn start:ci` into its own step so `yarn deps` and `yarn build` run before hitting the migration step.